lib/generators/manageiq/provider/templates/systemd/%plugin_name%_%manager_path%_event_catcher@.service

```
@@ -5,7 +5,9 @@ WantedBy=<%= plugin_name %>_<%= manager_path %>_event_catcher.target
 [Service]
 WorkingDirectory=/var/www/miq/vmdb
 Environment=BUNDLER_GROUPS=manageiq_default,ui_dependencies
-ExecStart=/bin/bash -lc 'exec ruby lib/workers/bin/run_single_worker.rb <%= class_name %>::<%= manager_type %>::EventCatcher --heartbeat --guid=%i'
+EnvironmentFile=/etc/default/manageiq.properties
+EnvironmentFile=/etc/default/manageiq-productization.properties
```

According to the docs you can pass a wildcard, so can we do

```suggestion
EnvironmentFile=/etc/default/manageiq*.properties
```